rotkehlchen/db/upgrades/v36_v37.py

```diff
@@ -213,13 +214,52 @@ def _fix_kraken_eth2_events(write_cursor: 'DBCursor') -> None:
     log.debug('Exit _fix_kraken_eth2_events')
 
 
+def remove_ftx_data(write_cursor: 'DBCursor') -> None:
+    """Removes FTX-related settings from the DB"""
+    write_cursor.execute(
+        'DELETE FROM user_credentials WHERE location IN (?, ?)',
+        (Location.FTX.serialize_for_db(), Location.FTXUS.serialize_for_db()),
+    )
+    write_cursor.execute(
+        'DELETE FROM user_credentials_mappings WHERE credential_location IN (?, ?)',
+        (Location.FTX.serialize_for_db(), Location.FTXUS.serialize_for_db()),
+    )
+    write_cursor.execute(
+        'DELETE FROM used_query_ranges WHERE name LIKE ? ESCAPE ?;',
+        (f'{str(Location.FTX)}\\_%', '\\'),
+    )
+    write_cursor.execute(
+        'DELETE FROM used_query_ranges WHERE name LIKE ? ESCAPE ?;',
+        (f'{str(Location.FTXUS)}\\_%', '\\'),
+    )
+    non_syncing_exchanges_in_db = write_cursor.execute(
+        'SELECT value FROM settings WHERE name="non_syncing_exchanges"',
+    ).fetchone()
+    if non_syncing_exchanges_in_db is not None:
+        ftx_locations = (Location.FTX.serialize(), Location.FTXUS.serialize())
+        non_syncing_exchanges = json.loads(non_syncing_exchanges_in_db[0])
+
+        ids_to_delete = []
+        for idx, exchange in enumerate(non_syncing_exchanges):
+            if exchange['location'] in ftx_locations:
+                ids_to_delete.append(idx)
+
+        for idx in reversed(ids_to_delete):  # reverse so that we don't mess up the indices
+            del non_syncing_exchanges[idx]
```

Comment:
This is too complicated a logic for what you need ...

```suggestion
        new_value = []
        for entry in non_syncing_exchanges:
            if entry['location'] not in (Location.FTX.serialize(), Location.FTXUS.serialize()):
                new_values.append(entry)
```